recipes/emboss/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: emboss
+  version: "6.5.7"
+
+build:
+  number: 0
+  skip: False
+
+source:
+  fn: EMBOSS-6.5.7.tar.gz
+  url: ftp://emboss.open-bio.org/pub/EMBOSS/old/6.5.0/EMBOSS-6.5.7.tar.gz
+  md5: 6a2cb3f93d5e9415c74ab0f6b1ede5f0
+
+requirements:
+  build:
+    #
+  run:
+    # 
```

Comment:
If there are really no dependencies, you can remove the entire requirements section.